.connector-store/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Salesforce Marketing Cloud",
+    "owner": "WSO2",
+    "category": "Sales and Marketing",
+    "documentationUrl": "https://mi.docs.wso2.com/en/latest/reference/connectors/salesforce-marketing-cloud-connector/salesforce-marketing-cloud-overview/",
+    "description": "The Salesforce Marketing Cloud Connector allows you to access the Salesforce Marketing Cloud Engagement APIs, enabling you to manage journeys, contacts, and marketing campaigns efficiently.",
+    "status": "Active",
+    "releases": [
+        {
+            "tagName": "v1.0.0",
+            "products": [
+                "MI 4.3.0",
```

Comment:
The new response model does not work in MI 4.3.0. So, we cannot add 4.3.0 as a supported product. For now lets remove MI 4.3.0 as a supported version.